src/main/java/com/faforever/api/leaderboard/LeaderboardService.java

```diff
@@ -35,8 +37,10 @@ public LeaderboardService(GlobalLeaderboardRepository globalLeaderboardRepositor
     return globalLeaderboardRepository.getLeaderboardByPage(getPageable(page, pageSize));
   }
 
-  public GlobalLeaderboardEntry getGlobalEntry(int playerId) {
-    return globalLeaderboardRepository.findByPlayerId(playerId);
+  public GlobalLeaderboardEntry getGlobalEntry(String playerId) {
+    if(DataTypeValidation.isNumeric(playerId))
```

Comment:
Maybe with Brackets. I actually just read your guidelines and I can not find anything on that but we like to keep code style alike.

Reply:
Have added the brackets for the `if-else` block.

Reply:
I am not able to find out what is causing the test coverage to drop.

Reply:
Well if u add more lines without writing tests the coverage increases so if u write new stuff(like u did) test it. Make a unit test feed the Service a string and see if it throws the Exception u expected...

Reply:
https://reqtest.com/testing-blog/test-coverage-metrics/

Reply:
Got it. Will write tests and submit again.